pkg/api/scheme.go

```diff
@@ -77,6 +78,10 @@ var localSchemeBuilder = runtime.SchemeBuilder{
 var AddToScheme = localSchemeBuilder.AddToScheme
 
 func init() {
+	// This is used to add the List object type for outputing multiple input objects
+	coreGroupVersion := schema.GroupVersion{Group: "", Version: runtime.APIVersionInternal}
+	Scheme.AddKnownTypes(coreGroupVersion, &metainternalversion.List{})
```

Comment:
The comment here in the context of this file/package is a little confusing. Perhaps note that this is required for the `ctl` command?

Do we want to consider using a scheme that's internal to the `ctl` package or something instead too? It may make it easier for us to track what's registered.

Reply:
Happy to add a comment.
Would having a separate scheme open up to use missing adding things in future?

Reply:
I doubt it - tests should ensure that we're properly registering required types IMO